### DISCUSSION TITLE
Scoring System using speed and accuracy

### DISCUSSION THREAD

**Discussion by allergicapple** [monkeytypegame/monkeytype#5828] [Ideas]
Hello there,

I would like to have some kind of scoring system which uses both speed and typing accuracy to determine the proficiency of the typist as I personally think speed alone is not a good metric. I know, I can use stop on word and other things to worsen my score when making a typing error but all this is not easily represented in the graphs and other places.

So I'd suggest something like this as an initial idea kick-off:
Score = (AccuracyInPercent - 0.96) * Wpm
If below 0, set to 0.

That way an accuracy below 96% is worth 0 points. It's just bad performance. After that it scales to a multiplicator of 1 for a flawless test.

If that feels too harsh, one could add a middle point where:
Accuracy 0-90% -> 0 Points
90%-96% -> 0% to 50% * Wpm
96%-100% -> 50% to 100% * Wpm
This gives the testers some kind of consolation price for motivation with a ramp up in the higher percent range. This also dampens the sharp falloff in that area.

With this graph I can judge better how my typing performance as wpm is really not the top priority but rather the overall proficiency of typing.

What do you think?

**Comment by Miodec** [comment]
Not sure about his. Speed is already dependent on accuracy so this seems like it would add an unnecessary number to the result screen.

**Comment by allergicapple** [comment]
But would you consider 82 wpm with 95% accuracy better or 80 wpm with 99%? That's the point of the question. If 80/99 is better it should reflect in a single metric, which is higher than 82/95.
